.github/workflows/test.yaml

```diff
@@ -72,6 +72,21 @@ jobs:
           TEST_WALLET_ADDRESS: ${{ secrets.TEST_WALLET_ADDRESS }}
           FUEL_TESTNET_NETWORK_URL: ${{ secrets.FUEL_TESTNET_NETWORK_URL }}
 
+  create-fuels-e2e:
```

Comment:
Also I suppose @arboleya may have to do this, but we would want this to be a required workflow on the repo as well correct?

Reply:
@Dhaiwat10 I can do it right after this goes to `master`.